package.json

```diff
@@ -35,8 +35,8 @@
     "/server-build"
   ],
   "dependencies": {
-    "@conform-to/react": "^0.6.1",
-    "@conform-to/zod": "^0.6.1",
+    "@conform-to/react": "0.6.3",
```

Comment:
@kentcdodds I'm not sure if there's a reason for not using pinned versions as its would provide stronger gaurentees that the application still works as expected, e.g. in this patch release the types actually changed so if we were to run an upgrade even though it wasn't specified, it would of resulted in a breaking change for this application.